docs/inputs.md

```diff
@@ -441,6 +448,7 @@ An array of strings (or objects with a `text` property, if you add the `property
 * **badge** - name of the icon (or a two-character string) that should be displayed in the simple list item when editing. Icon names can be anything from the [Material Design Icon Set](https://material.io/icons/), or you can use two initials
 * **allowRepeatedItems** - allow the same item more than once. defaults to false
 * **autocomplete** - object with autocomplete options. Currently this is just the key `list` where the value is the name of a list that Amphora knows about accessible via `/<site>/_lists/<listName>`. Example:
+* **modifyList** - options that allow the user to modify lists via Kiln. Setting `allowRemove` to true, enables an X in the `autocomplete` that allows the user to remove that item from the autocomplete list.
```

Comment:
Shouldn't this (or the related `allowRemove` argument itself) be specified under `autocomplete` (the line above)?